06_deploy_masters.sh

```diff
@@ -4,15 +4,23 @@ set -eux
 
 # Note This logic will likely run in a container (on the bootstrap VM)
 # for the final solution, but for now we'll prototype the workflow here
-
+instack="ocp/master_nodes.json"
 export OS_TOKEN=fake-token
 export OS_URL=http://ostest-api.test.metalkube.org:6385/
+
+# Clean previously env 
+for node in $(jq -r .nodes[].name ${instack}); do
+  openstack baremetal node undeploy $node
```

Comment:
We need some kind of guard against running this when nodes haven't been deployed, (e.g. first run), otherwise the script just exits on error

+ for node in '$(jq -r .nodes[].name ${instack})'
+ openstack baremetal node undeploy openshift-master-0
Node openshift-master-0 could not be found. (HTTP 404)
[notstack@host11 dev-scripts]$